src/interfaces/token.spec.ts

```diff
@@ -26,6 +26,7 @@ const vaultsTokensMock = jest.fn();
 const ironBankBalancesMock = jest.fn();
 const ironBankTokensMock = jest.fn();
 const sendTransactionMock = jest.fn();
+const populateApproveMock = jest.fn().mockResolvedValue(true);
```

Comment:
We should try and resolve to a more accurate value, in this case a `PopulatedTransaction`

Reply:
added here https://github.com/yearn/yearn-sdk/pull/283/commits/d6b5daa48ab825a9c68f896cc3b23eb6eb214d24